internal/pkg/cli/interfaces.go

```diff
@@ -91,8 +91,8 @@ type sessionProvider interface {
 	sessionFromRoleProvider
 }
 
-type webAppDescriber interface {
-	Describe() (*describe.WebAppDesc, error)
+type appDescriber interface {
```

Comment:
should this be just "describer" now?

Reply:
I thought we might have envDescriber in future but yeah you are right, even then it might share the same interface.